test/models/lorenz63.jl

```diff
@@ -27,9 +26,9 @@ function get_params(
     return P(; (; (Symbol(k) => v for (k, v) in model_params_dict)...)...)
 end
 
-struct Lorenz63Model{S <: Real, T <: Real}
+struct Lorenz63Model{S <: Real, T <: Real, I}
     parameters::Lorenz63ModelParameters{S, T}
-    integrators::Vector{<:SciMLBase.AbstractODEIntegrator}
+    integrators::Vector{I}
```

Comment:
Just to check I'm understanding correctly - this is to avoid the issue described by https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-fields-with-abstract-type?

Reply:
And I guess avoid the direct dependency on `SciMLBase`

Reply:
Main point was avoiding loading `SciMLBase` directly, but at the same time adding `I` as extra parameter [avoids the abstract container as field](https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-fields-with-abstract-containers), yes.